lib/Doctrine/ORM/Tools/ConvertDoctrine1Schema.php

```diff
@@ -170,9 +172,10 @@ private function convertColumns($className, array $model, ClassMetadataInfo $met
      * @param string|array      $column
      * @param ClassMetadataInfo $metadata
      *
-     * @return array
+     * @return mixed[]
      *
      * @throws ToolsException
+     *
```

Comment:
?

Reply:
That's me rejecting something badly, I will remove this.